README.md

```diff
@@ -17,11 +17,14 @@ publish a document end-to-end e.g.
 # Run whitehall rake plus any required dependencies (DBs)
 whitehall$ govuk-docker run rake
 
-# Start content-tagger rails plus a minimal backend stack
-content-tagger$ govuk-docker run --stack backend
+# Run whitehall rake plus a minimal app stack
+whitehall$ govuk-docker run --stack app rake
+
+# Start content-tagger rails plus a minimal app stack
+content-tagger$ govuk-docker startup
 
 # Start content-publisher rails plus an end-to-end stack
-content-publisher$ govuk-docker run --stack e2e
+content-publisher$ govuk-docker startup e2e
```

Comment:
Will that work?  Is the e2e stack actually called "app-e2e"?

Reply:
Although it's not currently the case, we will be renaming it to that. See https://github.com/alphagov/govuk-docker/pull/60.

Reply:
:+1: I misread slack and thought we were going with "lite", "app" + variations, and "e2e"